stdlib/InteractiveUtils/src/help.jl

```diff
@@ -0,0 +1,15 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+"""
+This singleton struct provides the help "command" during interactive mode which
+redirects the user to the "?" help REPL mode.
+"""
+struct Help
+end
+
+const help = Help()
+Base.show(io::IO, m::MIME"text/plain", ::Help) = show(io, m, Base.Docs.parsedoc(Base.Docs.keywords[:help]))
+Base.show(io::IO, ::Help) = show(io, Base.Docs.parsedoc(Base.Docs.keywords[:help]))
```

Comment:
Maybe
```suggestion
Base.show(io::IO, m::MIME"text/plain", h::Help) = show(io, m, h()))
Base.show(io::IO, h::Help) = show(io, h()))
```
?

Reply:
Sure, but I think you have an extra parentheses there.

```suggestion
Base.show(io::IO, m::MIME"text/plain", h::Help) = show(io, m, h())
Base.show(io::IO, h::Help) = show(io, h())
```